.github/workflows/tests-linux.yml

```diff
@@ -0,0 +1,32 @@
+on: [push, pull_request]
+name: "tests/linux"
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.x, 1.13.x]
```

Comment:
Don't we require Go 1.14 at least?

Reply:
still says 13 here: https://github.com/perkeep/perkeep/blob/7321b90b6e112e7847cb6f3f9683c74d545d5059/make.go#L901